src/main/scala/ch/epfl/scala/sbtmissinglink/MissingLinkPlugin.scala

```diff
@@ -102,7 +102,8 @@ object MissingLinkPlugin extends AutoPlugin {
     missinglinkIgnoreSourcePackages := Nil,
     missinglinkIgnoreDestinationPackages := Nil,
     missinglinkExcludedDependencies := Nil,
-    concurrentRestrictions += Tags.limit(missinglinkConflictsTag, 1),
+    // unlimited (in practice) number of projects can be analysed concurrently
+    concurrentRestrictions += Tags.limit(missinglinkConflictsTag, 1024),
```

Comment:
Shouldn't we simply remove the setting?

Reply:
Many people will still find it helpful
```scala
// Make it easy to throttle the concurrency of running missing-link on multiple projects, it consumes a lot of memory
```

Reply:
I'm not saying to remove the Tag and its association to the task. But to remove the setting that sets `concurrentRestrictions` on it. Let people set their own `concurrentRestrictions` if they so wish.

Reply:
Gotcha! :wink: